project/till22G/README.rst

```diff
@@ -0,0 +1,132 @@
+========================
+Monte Carlo Integration
+========================
+
+Basic Information
+=================
+
+Monte Carlo Methods or Monte Carlo experiments are based on repeated random sampling.  They can be applied to complicated deterministic problems like optimization,  simulations or integration, which would otherwise be infeasible to compute in an acceptable time. Using random sampling simplifies these problems significantly.
+
+Problem to Solve
+==================
+
+In a lot of scenarios using integration methods like Trapezoid or
+Simpsons rule are more efficient than Monte Carlo integration. Furthermore
+these methods produce deterministic results, which are a lot easier to work
+with compared to random ones. On the downside these methods suffer from the curse
+of dimensionality. Therefore Monte Carlo Integration (and Monte Carlo methods in general)
+are usually used, when we decide that random approximations are better
+than forgoing results altogether. The power of the Monte Carlo estimator
+comes from its incredible simplicity.
+
+
+Challenges:
+
+- The biggest challenge is have a good random number generator, the goal is to use all little random points as possible to get a good estimate
+- Another challenge in this project is to define a suitable testing strategy, since the estimator does not produce deterministic results
+
+Prospective Users
+=================
+
+Prospecitive users are generally consists of people who want to calculate
+integrals and can accept the uncertainty in the results of Monte Carlo
+Integration. It is expected that users are experienced in coding, thus there will
+be no GUI, but a only an API.
+
+System Architecture
+===================
+
+System Architecture will be done during the project
+
+API Description
+===============
+
+This API description is subject to change.
+
+Python API:
+
+- class: MonteCarloIntegration
```

Comment:
It's a good start to write down at least one class.  Having more will be better.  Just don't hesitate to change.

Reply:
I fully agree that this is the better approach, but considering my other lectures (midterm exam are coming up), investing the time in studying for those sadly yields a higher "payoff" regarding points/grades, compared to starting with a prototype implementation 10 days earlier. Especially since I am only able to work minimally on the project during this time anyway. I will start prototyping as soon as possible after the exams. I hope this is understandable.